build/release/tmpl/general_incubator_vote_result.tmpl

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Title: [RESULT][VOTE] Release Apache Kyuubi(Incubating) {release_version} {release_rc_no}
+
+Content:
+Hi all,
+
+Thanks for reviewing and voting for Apache Kyuubi(Incubating)
+{release_version} {release_rc_no} release, I am happy to announce
+the release voting has passed with
+
+(* = binding)
++1:
+- {AA}*
+- {BB}
+- {CC}*
+- {DD}*
+
++0: None
+
```

Comment:
seems we can remove the empty line

Reply:
Keep it consistent, see https://www.mail-archive.com/dev@spark.apache.org/msg27936.html